bridge/discord/handlers.go

```diff
@@ -279,3 +287,14 @@ func handleEmbed(embed *discordgo.MessageEmbed) string {
 
 	return result
 }
+
+func (b *Bdiscord) handleDownloadFile(rmsg *config.Message, m *discordgo.MessageCreate) {
+	for _, attach := range m.Attachments {
+		data, err := helper.DownloadFile(attach.URL)
+		if err != nil {
+			b.Log.Errorf("download %s failed %#v", attach.URL, err)
+		}
+
```

Comment:
I tested the pull request and had the issue that images were not being displayed on Element web. The image name still contained the parameters of the image url. I'd maybe add code to remove the parameters from an url before using it as filename.

```go

		urlClean, _, _ := strings.Cut(attach.URL, "?")

		helper.HandleDownloadData(b.Log, rmsg, path.Base(urlClean), rmsg.Text, attach.URL, data, b.General)
```